02_ml_models/README.md

```diff
@@ -0,0 +1,77 @@
+* [ml5.js: Image Classification with MobileNet - video tutorial](https://youtu.be/yNkAuWz5lnY?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Webcam Image Classification - video tutorial](https://youtu.be/D9BoBSkLvFo?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Transfer Learning with Feature Extractor - video tutorial](https://youtu.be/kRpZ5OqUY6Y?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Feature Extractor Classification - video tutorial](https://youtu.be/eeO-rWYFuG0?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Feature Extractor Regression - video tutorial](https://youtu.be/aKgq0m1YjvQ?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+
+### ml5.js resources
+* [ml5 website](https://ml5js.org)
+* [ml5 web editor examples](https://github.com/ml5js/ml5-examples/)
+
+### ml5 code editor examples
+* [Drag and Drop Image Classification](https://editor.p5js.org/ml5/sketches/S1PLinE27)
+* [Webcam Image Classification](https://editor.p5js.org/ml5/sketches/IlF1JFvWjc)
+* [Webcam Image Classification Text to Speech](https://editor.p5js.org/ml5/sketches/Ry7EL4JvA3)
+* [Image Classification Multiple Images](https://editor.p5js.org/ml5/sketches/f3rqIqNey5)
+* [Image Classification Scavenger Hunt](https://editor.p5js.org/ml5/sketches/APzpeXOuEQ)
+
+### Defining Machine Learning yet again
+* "A computer program is said to learn from experience E with respect to some class of tasks T and performance measure P, if its performance at tasks in T, as measured by P, improves with experience E." -- Tom Mitchell (1998): [Machine Learning book](http://amzn.to/2nLdRgQ).
+  * Example: classifying images of dogs and cats.
+    * E = Watching you classify images as dogs or cats.
+    * T = Classifying images as dogs or cats.
+    * P = The % of images correctly classified.
+
+### Classification and Regression
+* Classification and regression both involve making a "prediction" based on input data.
+* Classification refers to predicting an output with a discrete set of possibilities like a set of categories or labels. For example: "Given an input image, is it a dog or cat?"
+* Regression refers to predicting an "continuous" output (a fancy way of saying number). For example: "Given the number of bedrooms, what is the price of a house?" or "Given an input image of a cat, how much does the cat weigh?"
+
+### Assignment:
+1. Watch [But what *is* a Neural Network?](https://youtu.be/aircAruvnKk) by 3Blue1Brown
+2. Read Andrey Kurenkov's ['Brief' History of Neural Nets and Deep Learning](http://www.andreykurenkov.com/writing/a-brief-history-of-neural-nets-and-deep-learning/)
+3. Read [ImageNet: The Data That Transformed AI Research—and Possibly the World](https://qz.com/1034972/the-data-that-changed-the-direction-of-ai-research-and-possibly-the-world/) by Dave Gershgorn (Note: Fei-Fei Li is no longer at Google; she is currently Co-Director of the Stanford Human-Centered AI Institute)
+4. Explore [ImageNet](http://image-net.org/index)
```

Comment:
What surprises you about this data set? What questions do you have? Thinking back to last week’s assignment, can you think of any ethical considerations around how this data was collected? Are there privacy considerations with the data?

Reply:
I liked the simple explanation from Kyle MacDonald's talk: "Machine learning is programming with examples, not instructions". Of course coming up with the model's themselves is not trivial, it does drive home the point that ML relies data. 

<img width="1524" alt="Screen Shot 2019-08-30 at 13 28 47" src="https://user-images.githubusercontent.com/3622055/64496103-87058700-d26f-11e9-9449-9c4f702335a7.png">

Reply:
Thanks! I've added both of these in b8ab0346f0a4c95eb973436499aebbaab647da6a!